ruby-gem/Rakefile

```diff
@@ -13,21 +13,53 @@ end
 def build
   test_server_template_dir = File.join(File.dirname(__FILE__), 'test-server')
 
+  # Ant precondition
+  calabash_js_dir = File.join(test_server_template_dir, 'calabash-js/src')
+  fail "#{calabash_js_dir} does not exist" unless File.directory?(calabash_js_dir)
+  # end Ant precondition
+
   Dir.mktmpdir do |workspace_dir|
 
-    @test_server_dir = File.join(workspace_dir, 'test-server')
+    test_server_dir = File.join(workspace_dir, 'test-server')
     FileUtils.cp_r(test_server_template_dir, workspace_dir)
 
+    # Ant staging
+
+    # Try munging the workspace copy of instrumentation_backend directly,
+    # instead of having Ant's 'staging' copy in addition(!)
+
+    # Not sure we still need these with gradle building...
+    #  <fileset erroronmissingdir="false" dir="${env.ANDROID_HOME}/add-ons/addon-google_apis-google_inc_-${android.api.level}/libs"/>
+    #  <fileset erroronmissingdir="false" dir="${env.ANDROID_HOME}/add-ons/addon-google_apis-google-${android.api.level}/libs"/>
```

Comment:
We need google apis because we have steps attempting to use GoogleMaps. Please check.